packages/node_modules/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -400,6 +400,76 @@ describe('plugin-mercury', () => {
             });
         });
       });
+
+      describe('Websocket proxy agent', () => {
+        it('connects to Mercury using proxy agent', () => {
+          const testProxyUrl = 'http://proxyurl.com:80';
+
+          webex.config.defaultMercuryOptions = {agent: {proxy: {href: testProxyUrl}}};
+          const promise = mercury.connect();
+
+          assert.isFalse(mercury.connected, 'Mercury is not connected');
+          assert.isTrue(mercury.connecting, 'Mercury is connecting');
+          mockWebSocket.open();
+
+          return promise
+            .then(() => {
+              assert.isTrue(mercury.connected, 'Mercury is connected');
+              assert.isFalse(mercury.connecting, 'Mercury is not connecting');
+              assert.calledWith(socketOpenStub, sinon.match(/ws:\/\/example.com/), sinon.match.has('agent',
+                sinon.match.has('proxy', sinon.match.has('href', testProxyUrl))));
+            });
+        });
+
+        it('connects to Mercury without proxy agent', () => {
+          const promise = mercury.connect();
```

Comment:
I think you need to `delete webex.config.defaultMercuryOptions;` here so the config doesn't get passed through.

Reply:
Will clean up that config after each test to be on the safe side